.github/workflows/build.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - name: Build and install benchgc
         working-directory: build
         run: |
+          pip install py-cpuinfo
```

Comment:
please update the dependency in `setup.py`, not here

Reply:
I already added in `setup.py`, but it got build error in ci

Reply:
changed `py-cpuinfo`  to  `cpuinfo` in `setup.py`

Reply:
fixed